### DISCUSSION TITLE
How to move SearchBar into another place

### DISCUSSION THREAD

**Discussion by Siemienik** [facebook/docusaurus#4116] [Q&A] [answered]
Hi everyone, I feel pleased to have possibility ask you for help 😄 

I need to move SearchBar in a Navbar from right to the centre (as the last item with position:`left`).

## Actual:

![image](https://user-images.githubusercontent.com/2893099/106142802-38ad9c80-6172-11eb-92af-bf8e17c70d91.png)

## Expected:

![image](https://user-images.githubusercontent.com/2893099/106142958-6692e100-6172-11eb-9794-7e3e6c26b594.png)

I will be delighted for the help. I searched the documentation and some source code. However, I still haven't any idea how to do that.

**Comment by slorber** [comment, marked as answer]
Currently there's no option to configure the search bar location.

The only way is to swizzle the theme navbar component and customize the default navbar implementation (cf swizzle doc)

**Reply by janniks** [reply]
It's now possible to do this via the [navbar settings](https://docusaurus.io/docs/api/themes/configuration#navbar-search)

e.g. adding this as the first navbar item:
```js
{
  type: 'search',
  position: 'left',
}
```
